docs/_future/getting_started_future.md

```diff
@@ -10,66 +10,67 @@ permalink: /future/getting-started
 
----
+Follow the [Quickstart Guide](https://api.slack.com/future/quickstart) to get instructions on how to manually or automatically install the Slack CLI based on your operating system. You can skip any instructions related to installing Deno or creating an app using a Deno template.
 
-### Install dependencies {#install-dependencies}
+Once you've logged into the CLI using `slack login` and verified your login using `slack auth list`, you can proceed with the instructions in this guide.
 
-Once the CLI is set up, make sure your machine has the most recent version of [Node](https://nodejs.org/en/) installed. Depending on what operating system your machine uses, you can either install Node through a package manager (such as [Homebrew](https://brew.sh/) for Mac) or directly from the website.
-
----
+#### Installing Node {#installing-node}
 
-### Accept the Beta Terms of Service {#accept-tos}
+Make sure your machine has the most recent version of [Node](https://nodejs.org/en/) installed. Depending on what operating system your machine uses, you can either install Node through a package manager (such as [Homebrew](https://brew.sh/) for Mac) or directly from the website.
 
-In order to use the next-generation platform features, you'll need to accept a Terms of Service for Slack Platform Beta. To do so, go to the Workspace Settings of the workspace you'll be developing with, then go to the Permissions tab. Find the "Slack Platform Beta" heading, then select the checkbox that says "I have read and agree to the Slack Platform Beta Service Terms" and save the section. Once you have accepted, the section will look like this:
+#### Accept the Beta Terms of Service {#accept-tos}
 
-![Beta Terms of Service](../assets/beta-tos-future.png "Beta Terms of Service")
+In order to use the next-generation platform features, you'll need to accept a Terms of Service (TOS) for Slack Platform Beta in your Workspace Settings [here](https://slack.com/admin/settings#hermes_permissionfs).
 
 > 💡 You must be an admin of your workspace to be able to access the Workspace Settings and accept the Terms of Service.
 
+If the TOS has been accepted, you will see this view under the "Slack Platform Beta" section when navigating to the above link:
+
+![Beta Terms of Service](../assets/beta-tos-future.png "Beta Terms of Service")
 
-Congratulations! You're now ready to start building using the [next-generation Platform](/bolt-js/future/getting-started#next-gen). 🎉 You can now proceed with either creating a new app by following the instructions below or [setting up an existing app](/bolt-js/future/setup-existing-app).
+Congratulations! You're now ready to start building using the [next-generation platform](/bolt-js/future/getting-started#next-gen). 🎉 You can now proceed with creating a new app by following the instructions below or explore how to [set up an existing app](/bolt-js/future/setup-existing-app) on the next-gen platform.
```

Comment:
```suggestion
Congratulations! You're now ready to start building using the [next-generation platform](/bolt-js/future/getting-started#next-gen). 🎉 You can now proceed with creating a new app by following the instructions below or explore how to [set up an existing app](/bolt-js/future/setup-existing-app).
```
Thoughts on removing the second mention of next-gen if it's mentioned in the sentence above?